src/api_routes_v4.rs

```diff
@@ -355,7 +356,8 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
             scope("/block")
               .route("/person", post().to(user_block_person))
               .route("/community", post().to(user_block_community))
-              .route("/instance", post().to(user_block_instance)),
+              .route("/instance", post().to(user_block_instance))
+              .route("/post_keywords", post().to(user_block_keyword_for_posts)),
```

Comment:
This can be removed, as the keywords to block will be included as part of `save_user_settings`